### PR TITLE
erikre-docs-25280259

### PR DESCRIPTION
**Pull request by Erikre**
## Why

- Closes #[Issue Number]

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by DanielEmerson — CHANGES_REQUESTED** [review]
It won't let me edit the file for some reason.  I'm leaving comments inline for the changes to transition MAM to WPJ.

File: windows/client-management/implement-server-side-mobile-application-management.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+WIP uses Mobile Application Management (MAM). MAM enrollment is integrated with adding a work account flow to a personal device. If both MAM and Azure AD-integrated MDM services are provided in an organization, a user's personal devices are enrolled to MAM or MDM, depending on the user's actions. If a user adds their work or school Azure AD account as a secondary account to the machine, their device is enrolled to MAM. If a user joins their device to Azure AD, it's enrolled to MDM. In general, a device that has a personal account as its primary account is considered a personal device and should be enrolled to MAM. An Azure AD join, and enrollment to MDM, should be used to manage corporate devices.
```
Comment (DanielEmerson) [inline windows/client-management/implement-server-side-mobile-application-management.md:18]: "WIP uses Workplace Join (WPJ).  WPJ is integrated..."
Delete second sentence "If both..."
"If a user adds their work or school Entra ID account as a secondary account to the machine, their device registered with WPJ."
".. personal account as its primary account is considered a personal device and should be registered with WPJ."

File: windows/client-management/implement-server-side-mobile-application-management.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+WIP takes advantage of [built-in policies](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip) to protect company data on the device. To protect user-owned applications on personal devices, MAM limits enforcement of WIP policies to [enlightened apps](/windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip) and WIP-aware apps. Enlightened apps can differentiate between corporate and personal data, correctly determining which to protect based on WIP policies. WIP-aware apps indicate to Windows that they don't handle personal data, and therefore, it's safe for Windows to protect data on their behalf.
```
Comment (DanielEmerson) [inline windows/client-management/implement-server-side-mobile-application-management.md:26]: Replace MAM with WPJ

File: windows/client-management/implement-server-side-mobile-application-management.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 
-MAM enrollment is integrated with adding a work account flow to a personal device. If both MAM and Azure AD-integrated MDM services are provided in an organization, a user's personal devices are enrolled to MAM or MDM, depending on the user's actions. If a user adds their work or school Azure AD account as a secondary account to the machine, their device is enrolled to MAM. If a user joins their device to Azure AD, it's enrolled to MDM. In general, a device that has a personal account as its primary account is considered a personal device and should be enrolled to MAM. An Azure AD join, and enrollment to MDM, should be used to manage corporate devices.
+WIP uses Workplace Join (WPJ). WPJ is integrated with adding a work account flow to a personal device. If a user adds their work or school Entra ID account as a secondary account to the machine, their device registered with WPJ. If a user joins their device to Azure AD, it's enrolled to MDM. In general, a device that has a personal account as its primary account is considered a personal device and should be registered with WPJ. An Azure AD join, and enrollment to MDM, should be used to manage corporate devices.
```
Comment (DanielEmerson) [inline windows/client-management/implement-server-side-mobile-application-management.md:18]: "...their device is registered with WPJ."

File: windows/client-management/implement-server-side-mobile-application-management.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+WPJ takes advantage of [built-in policies](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip) to protect company data on the device. To protect user-owned applications on personal devices, MAM limits enforcement of WIP policies to [enlightened apps](/windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip) and WIP-aware apps. Enlightened apps can differentiate between corporate and personal data, correctly determining which to protect based on WIP policies. WIP-aware apps indicate to Windows that they don't handle personal data, and therefore, it's safe for Windows to protect data on their behalf.
```
Comment (DanielEmerson) [inline windows/client-management/implement-server-side-mobile-application-management.md:26]: First instance should stay WIP, "WIP takes advantage..." 
MAM should be replaced with WPJ, "...personal devices, WPJ limits the enforcement..."

**Comment by Thuyaaung939** [comment]
My Telephone number bill, 20000 kyats pay update help me .
09755986680

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**